### PR TITLE
timg: build with sixel support

### PR DESCRIPTION
**Pull request by aicynide**
- build with sixel (libsixel dependency)
- build with poppler needed for pdf (libcairo and popper dependency)
- build with rsvg needed for svg (libcairo and librsvg dependency)

### CODE REVIEW

File: packages/timg/build.sh [hunk 1]
```diff
@@ -3,15 +3,19 @@ TERMUX_PKG_DESCRIPTION="A terminal image and video viewer"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="1.6.0"
-TERMUX_PKG_SRCURL=https://github.com/hzeller/timg/archive/24d7727a4ba44b32fcdfb8e21fc258a4638fbbfe.tar.gz
-TERMUX_PKG_SHA256=5fe2a0f68dacfe367cc82ed5c3a21a9c1455ae78df1acedb7c02ed27ef502833
+TERMUX_PKG_REVISION="1"
+TERMUX_PKG_SRCURL=https://github.com/hzeller/timg/archive/refs/tags/v1.6.0.tar.gz
```
Comment (Biswa96) [inline packages/timg/build.sh:7]:
```suggestion
TERMUX_PKG_SRCURL=https://github.com/hzeller/timg/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
```

**Comment by Biswa96** [comment]
> Github actions failed to build because there was issue in timg, its fixed by the dev in latest commit.

You can import the patch file here.

File: packages/timg/build.sh [hunk 2]
```diff
@@ -3,15 +3,18 @@ TERMUX_PKG_DESCRIPTION="A terminal image and video viewer"
...
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="1.6.0"
-TERMUX_PKG_SRCURL=https://github.com/hzeller/timg/archive/24d7727a4ba44b32fcdfb8e21fc258a4638fbbfe.tar.gz
-TERMUX_PKG_SHA256=5fe2a0f68dacfe367cc82ed5c3a21a9c1455ae78df1acedb7c02ed27ef502833
-TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="ffmpeg, graphicsmagick, libc++, libjpeg-turbo, libexif, zlib"
+TERMUX_PKG_REVISION="1"
+TERMUX_PKG_SRCURL=https://github.com/hzeller/timg/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=9e1b99b4eaed82297ad2ebbde02e3781775e3bba6d3e298d7598be5f4e1c49af
+TERMUX_PKG_AUTO_UPDATE=trueTERMUX_PKG_DEPENDS="ffmpeg, graphicsmagick, libc++, libcairo, libdeflate, libjpeg-turbo, libexif, librsvg, libsixel, poppler, zlib"
```
Comment (TomJo2000) [inline packages/timg/build.sh:9]: > `libsixel` not found. Seriously? What kind of error is it?

The kind that happens if you don't put a newline here after.
`TERMUX_PKG_AUTO_UPDATE=true`

Comment (TomJo2000) [reply packages/timg/build.sh:9]: Ah `TERMUX_PKG_AUTO_UPDATE` isn't linted.
Guess that's another linter PR for me...

Comment (TomJo2000) [reply packages/timg/build.sh]: To be fair, the linter would have caught that if it was any other variable.

**Review by TomJo2000 — APPROVED** [review]
If at first you don't succeed,
try, try, try, try, try,
try, try, try, try, try,
try, try, try, try, try,
try, try, try, try, try,
try, try, try, try, try,
try again.
(It was 26 CI runs)

**Comment by TomJo2000** [comment]
Rebasing it to the latest commit real quick to ensure it's running against the new linter.

**Pull request merged**